src/main/java/seedu/notus/command/PinCommand.java

```diff
@@ -86,7 +87,13 @@ public String execute() {
             return Formatter.formatString(e.getMessage());
         }
 
-        return Formatter.formatString(note.getTitle() + " pinned: " + note.getPinnedString());
+        if (note.getPinned()) {
+            pinStatus = "Pinned";
+        } else {
+            pinStatus = "Unpinned";
+        }
```

Comment:
Why didn't you modify the getPinnedString to return this instead of Y/N?

Reply:
Thanks for pointing that out. I thought it was used elsewhere as well but I realised its not so I changed it

Reply:
Great! Approved for merging